setup.py

```diff
@@ -37,13 +37,14 @@
     'numpy',
     'packaging',
     'pillow',
-    'pydantic>=2',
+    'pydantic<=2',
```

Comment:
i don't think the configs will work with pydantic v1

too many breaking changes